db/seeds/CBDU1.json

```diff
@@ -6,7 +6,7 @@
   "isMainService": "yes",
   "onlyAMF": "no",
   "companyName": "Limited Company Seed",
-  "companyNo": 1234567,
+  "companyNo": "946107",
```

Comment:
This is `946107` whereas the others are `00946107` – is this intentional?

Might come in handy for testing the routing to new registrations due to changing the Companies House number (eg `946107` to `00946107` should be a valid change), but I don't know if you'd want a separate seed for that.

Reply:
Yes thought I'd throw one in without the leading zeros, so it should catch any issues with that.